glmnet/linear.py

```diff
@@ -137,6 +149,13 @@ def __init__(self, alpha=1, n_lambda=100, min_lambda_ratio=1e-4,
         self.random_state = random_state
         self.verbose = verbose
 
+        if any(self.lower_limits)>0 if isinstance(self.lower_limits, np.ndarray) else self.lower_limits>0:
+            raise ValueError("lower_limits must be non-positive")
+
+        if any(self.upper_limits)<0 if isinstance(self.upper_limits, np.ndarray) else self.upper_limits<0:
+            raise ValueError("upper_limits must be positive")
```

Comment:
Do you mind checking the line lengths and adding spaces around the operators here?